sigar_interface.go

```diff
@@ -167,3 +169,16 @@ func (self DiskIo) Delta(other DiskIo) DiskIo {
 		IoTimeMs: self.IoTimeMs - other.IoTimeMs,
 	}
 }
+
+type SystemInfo struct {
```

Comment:
Is this struct too tightly coupled to the output from `uname`? I've had trouble in the past where the Windows implementation had to be shoehorned into the linux structs, and it wasn't great.